addons/test_mail_full/tests/test_sms_performance.py

```diff
@@ -51,7 +51,7 @@ def setUp(self):
     def test_message_sms_record_1_partner(self):
         record = self.test_record.with_user(self.env.user)
         pids = self.customer.ids
-        with self.mockSMSGateway(), self.assertQueryCount(employee=21):  # test_mail_enterprise: 21
+        with self.mockSMSGateway(), self.assertQueryCount(employee=26):  # test_mail_enterprise: 26
```

Comment:
Where does those additional queries come from ?

Reply:
This is related to your question in `_postprocess_iap_sent_sms`. There was a missing notification that was added so it has to fetch data for it.

Reply:
1. read message
2. read ir_model based on model name provided by message (because of `model_name` in notif format)
3. search partner associated to notification (partner name for notif format)
4. insert bus